### ISSUE TITLE
CR: Sponsor request changes to voucher generator and export (short term)

### ISSUE THREAD

**Issue by GerbenBosschieter** [teamforus/general#810]
Learn more about change requests here: https://bit.ly/39CWeEE

[Support Ticket #3436]() (ask @GerbenBosschieter to see the e-mail) 

### Requested by:
- Nijmegen 

### Change description
Nijmegen would like to add the following changes to the voucher generator and the data in the export files.

1. Combine/merge the voucher & productvoucher list + export (busvoordeel and meedoen)
2. Add the list + export with the transaction date:
- Product voucher: Transaction date
- Budget voucher: First transaction date

**Comment by maxvisser** [comment]
@GerbenBosschieter for short term you can create a service request in ITSM for this combined export (if needed)

On long term I would first combine the pages into one generator; I would just always show the date of the first transaction. If not needed I would suggest leaving out the time (privacy by design)

**Comment by GerbenBosschieter** [comment]
@maxvisser oke, this was meerwerk request by nijmegen after a talk they had with @jamalv @maartenfv. I am only the messenger, not my request ;)

@maartenfv @jamalv this was after I asked what they exactly wanted

**Comment by maxvisser** [comment]
@maartenfv @jamalv you guys had the talk could you help whats needed / who is needed

**Comment by jamalv** [comment]
As I recall what is needed as a user story is:
- As a sponsor I would like the voucher export to include the date (not time) at which the first transaction took place so that I can see and report when the vouchers are being used

Is that correct @GerbenBosschieter @maartenfv?

**Comment by maartenfv** [comment]
@GerbenBosschieter Do you have an update on this issue?

**Comment by GerbenBosschieter** [comment]
@maartenfv no, you and jamal requested me to ask what they exactly want and if to check if development is needed so we could check if we could build this or ask for meerwerk. I've created this issue after I did that. So I was just the guy who delivered the information so you guys could decide what we could do with this request.

@jamalv not sure whats clear and why we again need confirmation of what they want. Isn't clear by the description of the change reuqest itself? 

1. Combine/merge the voucher & productvoucher list + export (busvoordeel and meedoen) (this is a bit bigger and I think not the most important)
2. Add the list + export with the transaction date:
- Product voucher: Transaction date + in use: yes or no
- Budget voucher: First transaction date + in use: yes or now 

@jamalv @maartenfv 
I already hesitated when you guys requested me to call with them caus I was afraid they would get any feedback/propsal after the inofrmation I would gather about this request. This is also what I mentioned and and I asked for clearity about what would happened next and what was expected from me. You guys made it clear that I only had to call, get the information, create CR. And the rest something you guys would manage.

No really happy to see the status of this CR.

**Comment by jamalv** [comment]
Something clearly went wrong here. Let's solve the problem first and then look what's needed to avoid this in the future. 

**For now I think what's needed:**
- [ ] ****Decision**:** on doing this - I assume the scope is a few hours max (maybe 5) - @maartenfv we could use the Nijmegen budget for this. Who decides that? Or we can do it from internal support budget. What do you guys think? @maartenfv or @GerbenBosschieter 
- [ ] **Development** wise I expect it can be planned for Alex in the current sprint. I could give it to him on Wednesday during refinement @jamalv 
- [ ] **Release**: according to the release planning the next release will be the 29th of June, that would mean this could go live then, or maybe if done from master it can be done sooner (when max is back from holiday he can confirm the release date) @maxvisser 
- [ ] **Communicate** back with Nijmegen about the planning (and optionally the choice to do it from their budget) @GerbenBosschieter was addressed in e-mail sent last week or @maartenfv since this has a deals component with their internal budget?

**Extra context:**
- We are also working on the API connection, when that one is implemented, the municipality will also have this info in their own systems.

Regarding the learning part; do we have any deals policy on this? (getting requirements, estimates, communicating back to customers, etc) And especially on what steps in the pipeline are there and how handoff is done? @maartenfv

**Comment by jamalv** [comment]
> Maandelijks exporteren wij een Excel bestand waarop de aanvragen Busvoordeelabonnement te zien zijn. Hierop is in de kolom ‘in use’ onder andere te zien óf klant het abonnement in gebruik heeft. Is het misschien mogelijk om in dit Excel bestand de datum van verzilvering toe te voegen i.p.v. alleen de informatie dat hij ‘in use’ is?

**Comment by jamalv** [comment]
wordt geplanned, gerben stuurt antwoord. plan is 29ste live

**Comment by jamalv** [comment]
will be planned in this afternoon during refiment (created the task)

**Comment by jamalv** [comment]
gerben heeft terugkoppeling gedaan - 29ste is prima, release later is ook goed. ze willen graag augustus of oktober beginnen.

**Comment by maartenfv** [comment]
Kan deze worden gesloten?
